doc/source/conf.py

```diff
@@ -61,6 +61,7 @@
     "matplotlib.sphinxext.plot_directive",
     "sphinx.ext.intersphinx",
     "sphinx.ext.coverage",
+    "sphinx_copybutton",  # helps add a little “copy” button to the code blocks
```

Comment:
I don't think the comment adds much to this line (I think `copybutton` is fairly clear)